bundles/org.openhab.binding.fsinternetradio/src/main/java/org/openhab/binding/fsinternetradio/internal/handler/FSInternetRadioHandler.java

```diff
@@ -45,21 +47,27 @@
  * @author Mihaela Memova - removed the unused boolean parameter, changed the check for the PIN
  * @author Svilen Valkanov - changed handler initialization
  */
+@NonNullByDefault
 public class FSInternetRadioHandler extends BaseThingHandler {
 
     private final Logger logger = LoggerFactory.getLogger(FSInternetRadioHandler.class);
 
-    FrontierSiliconRadio radio;
+    public @Nullable FrontierSiliconRadio radio;
```

Comment:
I don't think there is any reason to expose this even more than package-private:
```suggestion
    @Nullable
    FrontierSiliconRadio radio;
```

?